Sources/Helpers/OAAppSettings.m

```diff
@@ -2248,7 +2248,9 @@ - (instancetype) init
         [_registeredPreferences setObject:_arrowsOnMap forKey:@"show_arrows_to_first_markers"];
         _directionLines = [OAProfileBoolean withKey:mapDirectionLinesKey defValue:YES];
         [_registeredPreferences setObject:_directionLines forKey:@"show_lines_to_first_markers"];
-
+        
+        _shouldShowWhatsNew = NO;
```

Comment:
All setting properties are stored in the UserDefaults and have a key assigned to them and the default value (see all other BOOL properties in this class.